text/000-buildpacks-contribute-default-process-type.md

```diff
@@ -127,8 +135,30 @@ The lifecycle if not provided any `-process-type` will set the default process t
 [unresolved-questions]: #unresolved-questions
 
 - Should `pack` continue to pass `-process-type web`?
-  - It could inspect the buildpack API and only pass the flag if the API version is below 0.X.
-  - Builders could set an environment variable to communicate to `pack` that it doesn't need to pass -process-type to the lifecycle
+
+  Options discussed:
+
+  A. `pack` stops sending `-process-type web` when `CNB_PLATFORM_API` is greater than `0.X`. The lifecycle will assume `default = true` for `web` processes from buildpacks implementing earlier buildpack APIs. The lifecycle could print a warning when buildpacks do not specify a process type. `-process-type <unknown process type>` could then become a failure case (vs. warn, what it is today).
```

Comment:
I'm really in favor of this option. How can we carry it forward? Can we incorporate it into this PR? Add it to the next version of the spec? Or create a new RFC for just that change?

@ekcasey any preference?

Reply:
It's also worth noting that this isn't solely for `pack` but would also be useful for any other platform.

Reply:
I agree
https://github.com/buildpacks/rfcs/pull/110#discussion_r515111439 https://github.com/buildpacks/rfcs/pull/110#discussion_r510205574

Reply:
Migration plan covered here: https://github.com/buildpacks/rfcs/pull/110/files#diff-00a2fb620563b974f876628cc19e746a4a96606f9dea0fd58dc1bb13f010b4abR103-R114